apps/system/utils/netcmd.c

```diff
@@ -95,6 +96,85 @@ struct ifconfig_cmd_info_s {
 extern int netdb_main(int argc, char *argv[]);
 #endif
 
+uint8_t * getIp(void)
+{
+	struct ifreq *ifr;
+	struct sockaddr_in *sin;
+	struct sockaddr *sa;
+	struct ifconf ifcfg;
+	int ret = -1;
+	int fd;
+	int num_reqs = 3;
+	int num_nic = 0;
+	uint8_t *ip = NULL;
+	struct ifaddrs *ifa = NULL, *ifp = NULL;
+	ret = netlib_getifaddrs(&ifa);
+	if (ret < 0) {
+		NETCMD_LOGE(NTAG, "get ifaddrs fail\n");
+		return NULL;
+	}
+
+	fd = socket(AF_INET, SOCK_DGRAM, 0);
+	if (fd < 0) {
+		NETCMD_LOGE(NTAG, "fail %s:%d\n", __FUNCTION__, __LINE__);
+		return NULL;
+	}
+	memset(&ifcfg, 0, sizeof(ifcfg));
+	ifcfg.ifc_buf = NULL;
+	ifcfg.ifc_len = sizeof(struct ifreq) * num_reqs;
+	ifcfg.ifc_buf = malloc(ifcfg.ifc_len);
+	if (ioctl(fd, SIOCGIFCONF, (unsigned long)&ifcfg) < 0) {
+		perror("SIOCGIFCONF ");
+		goto DONE;
+	}
+	num_nic = ifcfg.ifc_len / sizeof(struct ifreq);
+	ifr = ifcfg.ifc_req;
+	int i = 0;
+
+	for (; i < num_nic; ifr++, i++) {
+		sin = (struct sockaddr_in *)&ifr->ifr_addr;
+		if ((sin->sin_addr.s_addr) == INADDR_LOOPBACK) {
+			NETCMD_LOG(NTAG, "Loop Back\t");
+		} else {
+			struct ifreq tmp;
+			strncpy(tmp.ifr_name, ifr->ifr_name, IF_NAMESIZE);
+			ret = ioctl(fd, SIOCGIFHWADDR, (unsigned long)&tmp);
+			if (ret < 0) {
+				NETCMD_LOGE(NTAG, "fail %s:%d\n", __FUNCTION__, __LINE__);
+			} else {
+				sa = &tmp.ifr_hwaddr;
+			}
+
+			ret = ioctl(fd, SIOCGIFFLAGS, (unsigned long)ifr);
+			if (ret < 0) {
+				NETCMD_LOGE(NTAG, "fail %s:%d\n", __FUNCTION__, __LINE__);
+			}
+		}
+		if ((strncmp(inet_ntoa(sin->sin_addr), LOCALHOST, sizeof(inet_ntoa(sin->sin_addr))))!= 0) {
+			ip = rtw_malloc(INET_ADDRSTRLEN + 1);
```

Comment:
In TizenRT common part, we don't allow board-specifc functions. 
I think there is no error with just `malloc`.
Could you check?